.github/workflows/cron_jerigon_zero_testing.yml

```diff
@@ -0,0 +1,102 @@
+name: Jerigon Zero Testing
+
+on:
+  # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 0 * * 0"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  jerigon_zero_testing:
+    name: Jerigon Zero Testing - Integration and Benchmarking
+    runs-on: zero-ci
```

Comment:
`zero-ci` is shared among other tasks. Ben created `zero-reg` - stronger, and for use only with benchmarks. We should just make sure that we schedule checks at different time.

Reply:
Note that `zero-reg` can span 50 jobs in parallel, we may want to limit it to 1 especially if its used for benchmarking.

Reply:
added concurrency

Reply:
using concurrency for now